.gitmodules

```diff
@@ -7,3 +7,12 @@
 [submodule "leveldb"]
 	path = leveldb
 	url = https://github.com/google/leveldb
+[submodule "grpc"]
+	path = grpc
+	url = https://github.com/grpc/grpc
+[submodule "etcd3-cpp"]
+	path = etcd3
+	url = https://github.com/graysonchao/etcd3-cpp
```

Comment:
Can you describe the origin of this & the difference of your repo vs. that?

Reply:
The etcd3-cpp module just wraps the etcd API and provides some utilities that are completely generic/have nothing to do with credis in particular. I want to continue working on it after this project because I haven't found a good library to use etcd v3 from c++.

Reply:
Ok.  `path = etcd3` is inconsistent with name of the repo, `etcd3-cpp`; any reason for that?  Can we make them consistent?